docs/diagrams/AddReminderCommandParserSeqDiagram.puml

```diff
@@ -29,7 +29,7 @@ create ReminderDescription
 ParserUtil -> ReminderDescription : new ReminderDescription(reminderDescription)
 activate ReminderDescription
 
-ReminderDescription --> ParserUtil : ReminderDescription
+ReminderDescription --> ParserUtil: reminderDescription
```

Comment:
II gave you some comments in telegram, you can refer to those and make some small edits, thanks!